plugins/zlux-agent/doc/swagger/fileapi.yaml

```diff
@@ -169,14 +169,168 @@ paths:
           schema:
             type: string
         '404':
           description: File could not be opened or does not exist
+    put:
+      summary: Creates a dataset
+      tags:
+        - Dataset
+      parameters:
+        - name: dataset
+          in: path
+          description: Cache results for future
+          required: true
+          type: string
+        - name: body
+          in: body
+          required: true
+          schema:
+            type: object
+            properties:
+              dsorg:
+                type: string
+                enum: [PS, PO]
+              blksz:
+                type: number
+              lrecl:
+                description: Desired record length
+                type: number
+              volser:
+                type: string
+              recfm:
+                type: string
+                enum: [A, B, F, U, V]
+              blkln:
+                description: The average data block length
+                type: number
+              ndisp:
+                type: string
+                default: CATLG
+                enum: [KEEP, CATLG]
+              strcls:
+                description: Desired SMS storage class
+                type: string
+              mngcls:
+                description: Desired SMS management class
+                type: string
+              datacls:
+                description: Desired SMS data class
+                type: string
+              space:
+                type: string
+                enum: [CYL, TRK]
+              dir:
+                type: number
+              prime:
+                type: number
+                description: Desired primary space quantity
+              secnd:
+                type: number
+                description: Desired secondary space quantity
+              avgr:
+                type: string
+                description: Specifies which allocation unit to be used
+                enum: [M, K, U]
+              dsnt:
+                type: string
+                description: Specifies which data set type to create
+                enum: [PDSE, PDS, HFS, EXTREQ, EXTPPREF, BASIC, LARGE]
+      consumes:
+        - application/json
+      produces:
+        - text/plain
+      responses:
+        '200':
+          description: Successfully created dataset
+          schema:
+            type: string
+        '400':
+          description: Invalid JSON
+          schema:
+            type: string
+          examples:
+            No JSON:
+              Cannot update file without JSON formatted record request
+            Invalid JSON:
+              Invalid JSON request body
+        '500':
+          description: Internal Server Error
+          schema:
+            type: string
+          examples:
+            DD Allocation Error:
+              Unable to allocate a DD for ACB
+            Unalloc error:
+              "error: ds unalloc dsn={dataset} dd=ddname, rc=rc sysRC=rc, sysRSN=rsn"
```

Comment:
Was this intended bracketing? [...] `{dataset} dd={ddname}` ?

Reply:
i dont think ddname resolves to a variable thats present in the swagger doc so no bracket.